### PR TITLE
Convert Create Community Modal to Page

### PR DESCRIPTION
**Pull request by true-eye**
image

### CODE REVIEW

**Review by gdjohnson — CHANGES_REQUESTED** [review]
Hey James, nice work. I managed to make a new offchain community and set some toggles. I left some minor comments about some wrapping divs and commented-out code.

We may want some kind of header or wrapping layout page, since right now, besides the "Back" button there's no way for the user to navigate out/away from the page. But that's a more high-level design issue that I'll leave to @zakhap's discretion.

Also, can we get a little more padding above the "Save changes" button?

Last, can we get the "Privacy" and "Invites" toggles to be inline with the text? The table's wide enough it should be possible, perhaps even with a simple `display: flex` on the wrapping `td` ele.

File: client/scripts/views/pages/create_community.ts [hunk 1]
```diff
@@ -81,7 +81,7 @@ const OffchainCommunityForm: m.Component<
       .map((_) => _.id)
       .filter((chain) => app.user.getAllRolesInCommunity({ chain }).length > 0);
 
-    return m('.compact-modal-body-max', [
+    return m('.class', [
```
[the remark this answers is not in the document]
Comment (zakhap) [reply client/scripts/views/pages/create_community.ts:84]: 👍 Whenever we return a component, the top-level classname should be representative of the Component name as well.

File: client/scripts/views/pages/create_community.ts [hunk 2]
```diff
@@ -1016,25 +1011,22 @@ interface CreateCommunityState {
...
+const CreateCommunity: m.Component<
   CreateCommunityAttrs,
   CreateCommunityState
 > = {
   oncreate: () => {
-    mixpanel.track('New Community', {
-      'Step No': 1,
-      Step: 'Modal Opened',
-    });
+    // mixpanel.track('New Community', {
```
[the remark this answers is not in the document]
Comment (zakhap) [reply client/scripts/views/pages/create_community.ts:1019]: Yes, just remove all the mixpanel stuff here

File: client/scripts/views/pages/create_community.ts [hunk 3]
```diff
@@ -267,7 +266,7 @@ const SubstrateForm: m.Component<SubstrateFormAttrs, SubstrateFormState> = {
     vnode.state.saving = false;
   },
   view: (vnode) => {
-    return m('.compact-modal-body-max', [
+    return m('.class', [
```
[the remark this answers is not in the document]
Comment (OliveSonDev) [reply client/scripts/views/pages/create_community.ts]: Plz check last commit.

File: client/scripts/views/pages/create_community.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (OliveSonDev) [reply client/scripts/views/pages/create_community.ts]: Plz check last commit.

[the remark this answers is not in the document]
Comment (true-eye) [reply client/scripts/views/pages/create_community.ts]: Plz check last commit.

File: client/scripts/views/pages/create_community.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (true-eye) [reply client/scripts/views/pages/create_community.ts]: Plz check last commit.

File: client/scripts/views/pages/create_community.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (true-eye) [reply client/scripts/views/pages/create_community.ts]: Plz check last commit.

File: client/scripts/views/pages/create_community.ts [hunk 4]
```diff
@@ -1016,25 +1016,16 @@ interface CreateCommunityState {
...
   oninit: (vnode) => {
     vnode.state.activeForm = CommunityType.OffchainCommunity;
   },
   view: (vnode: m.VnodeDOM<CreateCommunityAttrs, CreateCommunityState>) => {
-    return m('.CreateCommunityModal', [
-      m('.compact-modal-title', [
-        m('h3', 'New Commonwealth Community'),
-        m(CompactModalExitButton),
-      ]),
+    return m('div', { class: 'container CreateCommunity' }, [
```
Comment (zakhap) [inline client/scripts/views/pages/create_community.ts:1026]: One last change before merging this PR: 

- We should return this form inside of a Sublayout component like all of the other pages on CW. Linking to a blank page with just a form isn't going to cut it. 
  - Pass the attr `hideSidebar: true;` into Sublayout as this page isn't scoped within a community, so no need for the sidebar. (This should not impact the view of the SidebarSwitcher, just the Sidebar menu)

**Review by zakhap — CHANGES_REQUESTED** [review]

**Comment by zakhap** [comment]
After merging in an update to the create community modal, this PR is out of sync with master. You'll need to move over the new code from PR #1534.

**Review by jnaviask — APPROVED** [review]

**Review by jnaviask — COMMENTED** [review]
This is not ready to go, I'm gonna take over work on it for now in a new branch.

**Pull request closed without merging**